docs/howto/how_to_debug.md

```diff
@@ -171,7 +171,7 @@ ifconfig
 
 Assuming that the IP address of the industrial computer LAN is: `192.168.3.137`,
 open your Chrome or Firefox browser, enter the following URL:
-http://192.168.3.137:8888/, start the module (`Planning`) to be debugged as
+<http://192.168.3.137:8888/>, start the module (`Planning`) to be debugged as
 shown in
 [Start the module that needs debugging](#Start-the-module-that-needs-to-be-debugged)
 section.
```

Comment:
Start the module to debug.

Reply:
This doesn't match: `[Start the module that needs debugging](#Start-the-module-that-needs-to-be-debugged)`.